mentat/code_context.py

```diff
@@ -100,20 +94,27 @@ def display_context(self):
         else:
             stream.send(f"{prefix}Auto-Context: Disabled")
 
-    def display_features(self):
-        """Display a summary of all active features"""
-        session_context = SESSION_CONTEXT.get()
-        stream = session_context.stream
```

Comment:
I'm not sure if there's a clean way to do this but it might be nice to somehow have the features displayed in similarity order so you see what the model thinks is most likely to be useful first.